docs(dev): Tighten up CONTRIBUTING and its linkage

StoneyJackson opened the pull request:
* Add links in CONTRIBUTING.md to all files in docs/dev/*
* Tweek and expand various sections

Closes: #13

StoneyJackson committed:
docs(dev): Tighten up CONTRIBUTING and its linkage

* Add links in CONTRIBUTING.md to all files in docs/dev/*
* Tweek and expand various sections

Closes: #13

Signed-off-by: Stoney Jackson <dr.stoney@gmail.com>

kwurst commented on CONTRIBUTING.md:
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
 This project is a [LibreFoodPantry](https://librefoodpantry.org) project.
-We are a member of this larger community and shares its resources
-and policies including its
+We are a part of this larger community and shares its policies and resources.
```
I suggest "values, policies, and resources."

kwurst replied:
And "share", not "shares".

StoneyJackson replied:
Thanks! Fixing.

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
+will be publicly visible.
+
+Next, on GitHub or GitLab, __fork the project(s)__ you will be working on.
+If you are working in a shop, your shop manager will create a shop fork.
```
Maybe link to the shop setup documentation somewhere under here.

StoneyJackson replied:
Good idea. I'm adding it.

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
+* And credit all co-authors in the commit message
 
+Please read
+[Licensing](docs/dev/licensing.md)
```
This doesn't link to anything (no document named "licensing" in this folder).

StoneyJackson replied:
It should be there; I renamed docs/dev/copyright-licensing-and-dco.md → docs/dev/licensing.md . I was able to test this link by viewing the "code" tab and changing the branch to "contributing", then opening CONTRIBUTING.md and clicking on the link.

I'm marking this one resolved.

kwurst reviewed (changes requested):
I left one suggestion for a change. I feel like entirely removing Vision and Mission was a mistake. I think they can be subsumed under "values".

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -10,41 +10,91 @@ in the community are more likely to value and accept your contributions.
+it is more organized.
+
+For more information about our issue trackers, issue boards, issues, and labels
+please read [Issue Tracker](issue-tracker.md).
```
Doesn't link to anything, maybe:
docs/dev/issue-tracker.md

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -60,6 +110,8 @@ There are several types of issues in the issue tracker:
 Issues can and should be added at anytime during development.
 If you see something, say something.
 
+See also [Issue Tracker](issue-tracker.md).
```
Doesn't link, see previous comment above.

StoneyJackson replied:
This is real. Fixing it and the others.

StoneyJackson replied:
Thanks for catching it!

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -123,6 +177,8 @@ the following questions.
   How much effort is required by each strategy? Can the issue be broken into
   smaller more manageable issues? If so, what would those smaller issues be?
 
+See also [Issue Tracker](issue-tracker.md).
```
Doesn't link, see previous comment above.

cradkowski commented on CONTRIBUTING.md:
```diff
@@ -92,6 +144,8 @@ This is an ongoing process. When you look through the issue tracker and its
 boards looking for something to work on, plan to comment on issues and help
 prioritize, organized, and prepare them for work.
 
+See also [Issue Tracker](issue-tracker.md).
```
Doesn't link, see previous comment above.

StoneyJackson committed:
fix(contributing): Fix wording and broken links

Signed-off-by: Stoney Jackson <dr.stoney@gmail.com>

StoneyJackson committed:
refactor(contributing): Improve wording and style of shop-setup

Signed-off-by: Stoney Jackson <dr.stoney@gmail.com>

StoneyJackson committed:
fix(contributing): Restore Vision and Mission to CONTRIBUTING.md

Signed-off-by: Stoney Jackson <dr.stoney@gmail.com>

StoneyJackson commented:
Since I think I've addressed all of the issues you both raised (thanks again!), I'm going to use my superpowers to merge this PR. There are more changes I would like to make based on this, and I don't want to keep adding to this PR (since they are different issues).
